### PR TITLE
remove dynamic loading of node modules

### PR DESCRIPTION
**Pull request by asmega**
- fb-components-core is now a fixed packaged determined by the runner in the `package.json` manifest

### CODE REVIEW

File: lib/constants/constants.js [hunk 1]
```diff
@@ -126,17 +126,10 @@ if (PLATFORM_ENV) {
   }
 }
 
-// This value should be ~X.X.X-1
-// where X.X.X maps to version of govuk-frontend used by fb-components-core
-// -1 suffix allows latest published version matching ~X.X.X-*
-const COMPONENTS_VERSION = '~3.1.0-3'
-
 const CONSTANTS = Object.assign({
```
Comment (sequencemedialimited) [inline lib/constants/constants.js]: What's the benefit of `Object.assign()` here?

This looks to be just a literal, so
```
const CONSTANTS = {
  /* etc */
}
```
Should do

Comment (asmega) [reply lib/constants/constants.js]: i have no idea, this was already here

Comment (sequencemedialimited) [reply lib/constants/constants.js]: Kill it

File: lib/server/server-init.js [hunk 2]
```diff
@@ -5,10 +5,11 @@ const getRouter = require('./server-router')
 
 const initRunner = async (app, options = {}) => {
   const {
-    serviceSources,
-    locals
+    serviceSources
   } = await configureSources(options)
 
+  const locals = {govuk_frontend_version: '3.1.0'}
```
Comment (sequencemedialimited) [inline lib/server/server-init.js]: Odd that we have to manually change this :(

Comment (asmega) [reply lib/server/server-init.js]: i'll have a look to see if it can be removed. then we just use whatever comes thru the package.json

Comment (sequencemedialimited) [reply lib/server/server-init.js]: Do you know _why_ the app needs this?

Comment (sequencemedialimited) [reply lib/server/server-init.js]: Apparently, we were typing simultaneously

File: lib/constants/constants.js (hunk 1 above)
Comment (sequencemedialimited) [reply lib/constants/constants.js]: Current line `151` is also weird

Comment (asmega) [reply lib/constants/constants.js]: how? it's flattening a hash. 

151 is on a different line as it references itself

File: lib/server/server-init.js (hunk 2 above)
Comment (asmega) [reply lib/server/server-init.js]: i will leave this. its used to pull in the distribution of the css/js which is found in components core

File: lib/constants/constants.js (hunk 1 above)
Comment (sequencemedialimited) [reply lib/constants/constants.js]: Yeah, that's why it's weird

**Pull request merged**